### PR TITLE
Initial Bastion generator.

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: generators/app/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+module.exports = BastionBase.extend({
+    constructor: function () {
+        yeoman.generators.Base.apply(this, arguments);
+    },
+    promptUser: function () {
+        var done, prompts;
+
+        done = this.async();
+
+        console.log(this.yeoman);
```
Comment (waldenraines) [inline generators/app/index.js:13]: Is this intentional?

Comment (ehelms) [inline generators/app/index.js:13]: Yes.

File: CONTRIBUTING [hunk 2]
```diff
@@ -0,0 +1,35 @@
+## Contributing
+
+1. Fork the repo.
+
+2. Run the tests. We only take pull requests with passing tests, and
```
Comment (waldenraines) [inline CONTRIBUTING:5]: This project has (will have) tests?

Comment (ehelms) [inline CONTRIBUTING:5]: Added tests.

**Comment by waldenraines** [comment]
ACK (although I wish we didn't have to wrap things in IIFEs).

**Comment by ehelms** [comment]
That part is still up for debate.

**Comment by waldenraines** [comment]
Well I think I agree with the IIFEs and passing the functions into angular, I just wish we could wrap everything in a single IIFE at build time.

**Comment by thomasmckay** [comment]
ack after cursory testing (and dreams of all the features to come!)

**Pull request merged**